.github/workflows/tests.yml

```diff
@@ -97,7 +98,7 @@ jobs:
           custom_go_sdk_version: ${{ env.CURRENT_BRANCH_HEAD }}
           system_tests_branch: master
           network: ${{ env.NETWORK_URL }}
-          zbox_cli_branch: fix/upgrade_gosdk
+          zbox_cli_branch: staging
```

Comment:
this is part of https://github.com/0chain/gosdk/pull/335 which is not ready to be merged yet

Reply:
oh i see it was merged, line 81 will need changed to master and this will need merged https://github.com/0chain/actions/pull/7/files

Reply:
custom_go_sdk_version is unnecessary. We will get it worked on development branch.

Reply:
you're missing the purpose of this PR
The purpose is to merge a gosdk change with staging CLIs and check that it works 
This is to prevent gosdk being broken and left that way